### PR TITLE
Masthead

### PR DESCRIPTION
**Pull request by jgiardino**
https://jgiardino.github.io/patternfly-atomic/?p=viewall-components-masthead
https://jgiardino.github.io/patternfly-atomic/?p=viewall-components-navigation

Updates include:

- included skip to main link
- created separate components for nav, navlink, and utilities nav
- documentation and accessibility notes for masthead
- accessibility updates for masthead 

Still to do:

- write documentation for nav, navlink and utilities nav

Questions:

- Where/how should the skip to main link display? Where should it display for smaller screensizes? Is it even needed when the navigation is hidden and the bars icon displays?
- Keyboard access doesn't seem to work at all in patternlab components on Firefox.
- Why do the images not display on my github page? They display locally

### CODE REVIEW

File: source/_patterns/components/navigation/nav.hbs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<ul class="nav {{ulClass}}">
```
Comment (andresgalante) [inline source/_patterns/components/navigation/nav.hbs]: I think that `ulClass` and the other variables referring a type selector are too general. Is this a case to use [styleModifier](http://patternlab.io/docs/pattern-stylemodifier.html)? #51 

If it is I can ask Brian to fix our implementation so we can use it.

File: source/scss/_pf-masthead.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+.pf-masthead {
+  background-color: $pf-color-black-1000;
+  border-top: 3px solid $pf-color-blue-300;
+  color: $pf-color-black-300;
+  @include media-breakpoint-up(md) {
+    padding-left: ($grid-gutter-width-base / 2);
+  }
+}
+
+.pf-masthead__skip {
```
Comment (andresgalante) [inline source/scss/_pf-masthead.scss]: the skip link is creating horizontal scrolling

Comment (jgiardino) [inline source/scss/_pf-masthead.scss]: I can't believe I missed that. bitmoji

**Comment by andresgalante** [comment]
@jgiardino sorry for not been able to review this :( it's been a month!

I don't really understand `navlink.hbs` but I'll take a closer look during the weekend and merge it early next week.

Thanks a lot!

**Pull request merged**